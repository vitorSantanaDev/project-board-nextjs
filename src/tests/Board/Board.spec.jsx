import {} from '@testing-library/react'

describe('<Board />', () => {
  it('dummy test', () => {
    expect(1 + 2).toBe(3)
  })
})
