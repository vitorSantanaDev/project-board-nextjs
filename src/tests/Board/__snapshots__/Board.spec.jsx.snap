// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Board /> should match snapshot 1`] = `
.c0 {
  max-width: 1120px;
  border-radius: 6px;
  padding: 2rem;
  margin: 4rem auto;
  background-color: #17181f;
}

.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

.c2 {
  width: 90%;
  height: 50px;
  padding: 8px 16px;
  border-radius: 6px;
  border: 1px solid #424242;
  color: #fff;
  background-color: #20212c;
}

.c3 {
  width: 8%;
  height: 50px;
  cursor: pointer;
  border-radius: 6px;
  margin-left: 16px;
  background-color: #ffb800;
  -webkit-transition: ease-in-out 0.3s;
  transition: ease-in-out 0.3s;
}

.c3:hover {
  -webkit-transform: scale(1.05);
  -ms-transform: scale(1.05);
  transform: scale(1.05);
}

.c4 {
  color: #fff;
  margin-top: 2.5rem;
}

.c5 {
  margin: 2rem 0;
  padding: 1.2rem;
  border-radius: 6px;
  background-color: #20212c;
}

.c5 > p {
  color: #f2f6fc;
  font-size: 1.6rem;
  line-height: 150%;
  cursor: pointer;
}

.c6 {
  width: 100%;
  margin-top: 2rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
}

.c6 div {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

.c6 div time {
  color: #ffb800;
  margin-left: 0.5rem;
  margin-right: 1rem;
}

.c7 {
  border: none;
  background: transparent;
  cursor: pointer;
  margin-left: 1rem;
  -webkit-transition: ease-in-out 0.3s;
  transition: ease-in-out 0.3s;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

.c7:hover {
  opacity: 0.7;
  -webkit-transform: scale(1.05);
  -ms-transform: scale(1.05);
  transform: scale(1.05);
}

.c7 > span {
  color: #fff;
  margin-left: 0.7rem;
}

.c8 {
  border: none;
  background: transparent;
  cursor: pointer;
  margin-left: 1rem;
  -webkit-transition: ease-in-out 0.3s;
  transition: ease-in-out 0.3s;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

.c8:hover {
  opacity: 0.7;
  -webkit-transform: scale(1.05);
  -ms-transform: scale(1.05);
  transform: scale(1.05);
}

.c8 > span {
  color: #fff;
  margin-left: 0.7rem;
}

<main
  class="c0"
>
  <form
    class="c1"
  >
    <input
      class="c2"
      placeholder="Digite sua tarefa"
      type="text"
    />
    <button
      class="c3"
      type="submit"
    >
      <svg
        color="#17181f"
        fill="none"
        height="25"
        stroke="currentColor"
        stroke-linecap="round"
        stroke-linejoin="round"
        stroke-width="2"
        style="color: rgb(23, 24, 31);"
        viewBox="0 0 24 24"
        width="25"
        xmlns="http://www.w3.org/2000/svg"
      >
        <line
          x1="12"
          x2="12"
          y1="5"
          y2="19"
        />
        <line
          x1="5"
          x2="19"
          y1="12"
          y2="12"
        />
      </svg>
    </button>
  </form>
  <h1
    class="c4"
  >
    VocÃª tem 2 tarefas!
  </h1>
  <section
    class=""
  >
    <article
      class="c5"
      role="article"
    >
      <p>
        Aprender a criar projetos usando Next JS.
      </p>
      <div
        class="c6"
      >
        <div>
          <div>
            <svg
              color="#FFB800"
              fill="none"
              height="20"
              stroke="currentColor"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="2"
              style="color: rgb(255, 184, 0);"
              viewBox="0 0 24 24"
              width="20"
              xmlns="http://www.w3.org/2000/svg"
            >
              <rect
                height="18"
                rx="2"
                ry="2"
                width="18"
                x="3"
                y="4"
              />
              <line
                x1="16"
                x2="16"
                y1="2"
                y2="6"
              />
              <line
                x1="8"
                x2="8"
                y1="2"
                y2="6"
              />
              <line
                x1="3"
                x2="21"
                y1="10"
                y2="10"
              />
            </svg>
            <time>
              17 julho 2021
            </time>
          </div>
          <button
            class="c7"
          >
            <svg
              color="#FFF"
              fill="none"
              height="20"
              stroke="currentColor"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="2"
              style="color: rgb(255, 255, 255);"
              viewBox="0 0 24 24"
              width="20"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M17 3a2.828 2.828 0 1 1 4 4L7.5 20.5 2 22l1.5-5.5L17 3z"
              />
            </svg>
            <span>
              Editar
            </span>
          </button>
        </div>
        <button
          class="c8"
        >
          <svg
            color="#FF3636"
            fill="none"
            height="20"
            stroke="currentColor"
            stroke-linecap="round"
            stroke-linejoin="round"
            stroke-width="2"
            style="color: rgb(255, 54, 54);"
            viewBox="0 0 24 24"
            width="20"
            xmlns="http://www.w3.org/2000/svg"
          >
            <polyline
              points="3 6 5 6 21 6"
            />
            <path
              d="M19 6v14a2 2 0 0 1-2 2H7a2 2 0 0 1-2-2V6m3 0V4a2 2 0 0 1 2-2h4a2 2 0 0 1 2 2v2"
            />
          </svg>
          <span>
            Excluir
          </span>
        </button>
      </div>
    </article>
  </section>
</main>
`;
